hack/add-openshift-annotations.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+OPENSHIFT_VERSIONS="\"v4.11-v4.14\""
+
+{
+    echo ""
+    echo "  # OpenShift specific annotations"
+    echo "  com.redhat.openshift.versions: $OPENSHIFT_VERSIONS"
+} >> bundle/metadata/annotations.yaml
+
+echo "LABEL com.redhat.openshift.versions=$OPENSHIFT_VERSIONS" >> bundle.Dockerfile
```

Comment:
Is there really no way to get operator-sdk to set these for us?

Reply:
It seems not... I checked how other projects do because I expected something in the `operator-sdk`. And... it seems everybody is doing this 😓